packages/cli/test/integration/workflows/workflows.controller.test.ts

```diff
@@ -659,7 +659,15 @@ describe('GET /workflows', () => {
 
 			expect(response.body).toEqual({
 				count: 1,
-				data: [objectContaining({ name: 'First', tags: [{ id: any(String), name: 'A' }] })],
+				data: [
+					objectContaining({
```

Comment:
Updating this test to make sure the workflow includes all tags and not only the tags that matched the query. This was a bug in the existing implementation, but it's fixed now.